docs/user/alerting/stack-alerts/es-query.asciidoc

```diff
@@ -6,20 +6,20 @@ The ES query alert type is designed to run a user-configured {es} query over ind
 actions to run when the threshold condition is met.
 
 [float]
-==== Creating the alert
+==== Create the alert
 
-An ES query alert can be created from the *Create* button in the <<alert-management, alert management UI>>. Fill in the <<defining-alerts-general-details, general alert details>>, then select *ES query*.
+Fill in the <<defining-alerts-general-details, alert details>>, then select *ES query*.
 
 [role="screenshot"]
-image::user/alerting/images/alert-types-es-query-select.png[Choosing an ES query alert type]
+image::user/alerting/images/alert-types-es-query-select.png[Choose an ES query alert type]
 
 [float]
-==== Defining the conditions
+==== Conditions
 
 The ES query alert has 5 clauses that define the condition to detect.
```

Comment:
Not sure clause is the right word here.  Also, the number 5 confuses me because I see three sections in the image: index & size, es query, condition.

Reply:
Maybe follow a pattern where each section starts with a verb.

Define the ... to detect the condition.